botocore/__init__.py

```diff
@@ -60,8 +60,17 @@ def emit(self, record):
 
 BOTOCORE_ROOT = os.path.dirname(os.path.abspath(__file__))
 
+
 # Used to specify anonymous (unsigned) request signature
-UNSIGNED = object()
+class _UNSIGNED(object):
```

Comment:
Seems a bit odd, but then again there's no reason we need to keep the class around.